Dockerfile.build.aarch64-linux

```diff
@@ -0,0 +1,151 @@
+FROM nvidia/cuda:10.0-devel-ubuntu16.04
```

Comment:
Does it need to be nvidia/cuda:10.0-devel-ubuntu16.04 as we are installing Jetpack on our own anyway?

Reply:
I'm not sure if that package that we use for install provides the whole SDK or only the cross libraries.